contracts/registry/ReverseRegistrar.sol

```diff
@@ -116,4 +139,38 @@ contract ReverseRegistrar {
             ret := keccak256(0, 40)
         }
     }
+
+    /**
+     * @dev Transfers ownership of the reverse ENS record associated with the
+     *      calling account.
```

Comment:
Missing `@param addr`. Also can you change the parameter name to something more meaningful?

Reply:
maybe `controllerAddress` or something?